.gitignore

```diff
@@ -5,4 +5,5 @@ npm-debug.log
 .eslintcache
 .gatsby-context.js
 .idea
-gatsby-starter-lumen.iml
\ No newline at end of file
+gatsby-starter-lumen.iml
+package-lock.json
```

Comment:
We should not be ignoring `package-lock.json`, it's necessary for when we run `npm install`.